packages/server/src/controllers/gateway.controller.ts

```
@@ -1,9 +1,8 @@
 import { AgentResponse } from './agents.controller.js';
 import { AgentStorage } from '../agents.storage.js';
 import { AgentService } from '../services/agent.service.js';
-import { SupervisorService } from '../services/supervisor.service.js';
 import ServerError from '../utils/error.js';
-import { OnModuleInit } from '@nestjs/common';
+import { Controller, OnModuleInit, Post } from '@nestjs/common';
```

ESLint

[error] 'Controller' is defined but never used.

(@typescript-eslint/no-unused-vars)